src/treestatus/api/treestatus_api/api.py

```diff
@@ -20,6 +21,20 @@
 
 UNSET = object()
 TREE_SUMMARY_LOG_LIMIT = 5
+STATUSPAGE_URL = 'https://api.statuspage.io/v1'
```

Comment:
Shouldn't this be somewhere in `settings.py`?

Reply:
this is a constant url which we use across all channels